torch/csrc/jit/python/script_init.cpp

```diff
@@ -478,7 +478,8 @@ static void setInputTensorTypes(
     TORCH_INTERNAL_ASSERT(input_values.size() == param_count_list.size());
   }
   for (auto v : input_values) {
-    AT_ASSERT(s_iter != stack.end());
+    if (s_iter == stack.end())
```

Comment:
why the mismatch now?

Reply:
Needed this for debug. Reverted. Thanks.